website/template/base.twig

```diff
@@ -19,7 +19,7 @@
     </script>
 </head>
 <body>
-<div class="d-flex">
+<div class="d-flex no-mobile-x-scroll">
```

Comment:
When we open the menu, we should not be able to scroll horizontally. 

<img width="463" alt="Screenshot 2020-05-02 at 23 36 44" src="https://user-images.githubusercontent.com/1275206/80892904-0ac42600-8cce-11ea-9f52-f35e3382b850.png">